README.md

```diff
@@ -232,6 +235,46 @@ This collection of rules can be extended with custom rules using the `--append-r
 
 More information describing how rules are set up and an overview of all the Rules that are applied by this linter are documented [here](docs/rules.md).
 
+## Custom Rules
```

Comment:
@kddejong @miparnisari @Birds thoughts on terminology to distinguish from traditional custom rules?

```suggestion
## Concise Custom Rules
```

Reply:
Assuming the reason we'd consider rename is because technically there can be "Custom Rules" for the existing rule system. In that case I'd propose the following as something that doesn't even use the same "Custom" keyword. 

"Modular Rules"

Reply:
I'm okay with "custom rules"

Reply:
Stay the course with custom rules?

Reply:
Will resolve and leave as is under "Custom Rules" pending further opinions/thoughts